src/lib/__tests__/InstantSearch-test.js

```diff
@@ -421,17 +421,41 @@ describe('start', () => {
 
     expect(search.mainIndex.getHelper().state).toEqual(
       algoliasearchHelper.SearchParameters.make({
-        index: 'index_name',
+        index: 'indexName',
         hitsPerPage: 5,
         disjunctiveFacetsRefinements: { brand: ['Apple'] },
         disjunctiveFacets: ['brand'],
       })
     );
   });
 
+  it('forwards the `initialUiState` to the main index', () => {
```

Comment:
We don't test for nested indices?

Reply:
No we don't, it's already covered inside [`index`](https://github.com/algolia/instantsearch.js/blob/271fc7b2cd905d3ca388a5e31ecd2b4c2e3ca27e/src/widgets/index/__tests__/index-test.ts#L1379-L1435). From the InstantSearch perspective it's the same, it doesn't matter if there is 1 or 100 indices.